employee-accounts-web-app/src/pages/add.jsx

```diff
@@ -1,10 +1,123 @@
-import { Outlet} from "react-router-dom"
+
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { useState } from "react";
+import Alert from 'react-bootstrap/Alert';
 
 export function Add() {
-    return(
+  const [validated, setValidated] = useState(false);
+
+  const handleSubmit = (event) => {
+    const form = event.currentTarget;
+    if (form.checkValidity() === false) {
+      event.preventDefault();
+      event.stopPropagation();
+    }
+
+    setValidated(true);
+  };
+
+  function createList() {
+    console.log('clicked')
+    return (
```

Comment:
Currently the fields are just clearing when you click the button. I'm not sure what you wanted to happen here.